src/app-layout/visual-refresh/background.tsx

```diff
@@ -10,8 +10,11 @@ import styles from './styles.css.js';
  * that the design tokens used are overridden with the appropriate values.
  */
 export default function Background() {
-  const { hasNotificationsContent, hasStickyBackground, stickyNotifications } = useAppLayoutInternals();
-
+  const { hasNotificationsContent, hasStickyBackground, stickyNotifications, hasBreadcrumbs, isOverlapDisabled } =
+    useAppLayoutInternals();
+  if (hasNotificationsContent && !hasBreadcrumbs && isOverlapDisabled) {
```

Comment:
Per my other comment we can just check `!breadcrumbs` and save ourselves the additional property. I think I already have code doing that in other places.